docs/sources/alerting/alerting-rules/templates/_index.md

```diff
@@ -0,0 +1,215 @@
+```
+
+The result of the annotation would now be:
+
+```
+CPU usage for Instance 1 has exceeded 80% (81.2345) for the last 5 minutes.
+```
+
+### How to template an annotation
+
+To template an annotation:
+
+1. Navigate to **Alerts & IRM** -> **Alert rules** -> create or edit an **alert rule**.
+1. Scroll down to the **Configure notification message** section.
+1. Copy in your template in the corresponding annotation field (`summary`, `description`, `runbook_url`, `custom`)
+
+### Preview annotation templates
+
+You can template annotations when creating or editing an alert rule.
+
+{{< figure src="/media/docs/alerting/alert-rule-using-annotation-template.png" max-width="1200px" alt="An alert rule templating the annotation summary" >}}
+
+Two common methods are used to test and preview annotation templates:
+
+1. Trigger the alert and [view the alert instance state in the Grafana UI](ref:view-alert-state), where all annotations of the alert instance are displayed.
+1. Use a notification template that displays all annotations, then [preview the notification template](ref:preview-notifications) using the alert instance.
+
+## Template labels
+
+The set of [labels](ref:labels) for an alert instance is used to uniquely identify that alert among all other alert instances.
+
+Labels determine how alerts are routed and managed for notifications, making their design key to the effectiveness of your alerting system.
+
+Labels can be returned from an alert rule query, such as the `pod` label in a Kubernetes Prometheus query. You can also define additional labels in the alert rule to provide extra information for processing alerts.
+
+Like annotations, labels are key-value pairs that can contain plain text or template code evaluated when the alert fires.
+
+Template labels when the labels returned by your queries are insufficient. For instance:
+
+- A new label based on a query value can group a subset of alerts differently, changing how notifications are sent.
+- A new label based on a query value can be used in a notification policy to alter the notification contact point.
+
+Here’s an example of templating a `severity` label based on the query value:
+
+```
+{{ if (gt $values.A.Value 90.0) -}}
+critical
+{{ else if (gt $values.A.Value 80.0) -}}
+high
+{{ else if (gt $values.A.Value 60.0) -}}
+medium
+{{ else -}}
+low
+{{- end }}
+```
+
+In this example, the value of the `severity` label is determined by the query value, and the possible options are `critical`, `high`, `medium`, or `low`. You can then use the `severity` label to change their notifications—for instance, sending `critical` alerts immediately or routing `low` alerts to a specific team for further review.
+
+{{% admonition type="note" %}}
+You should avoid displaying query values in labels, as this may create numerous unique alert instances—one for each distinct label value. Instead, use annotations to inform about query values.
```

Comment:
```suggestion
You should avoid displaying query values in labels, as this may create numerous unique alert instances—one for each distinct label value. Instead, use annotations for query values.
```